src/clusterd/src/bin/clusterd.rs

```diff
@@ -239,6 +239,7 @@ async fn run(args: Args) -> Result<(), anyhow::Error> {
     emit_boot_diagnostics!(&BUILD_INFO);
 
     mz_alloc::register_metrics_into(&metrics_registry).await;
+    mz_ore::process::rusage::register_metrics_into(&metrics_registry).await;
```

Comment:
Should we add this to `environmentd` too, for good measure?

Reply:
I think that's what's happening below in `environmentd/main.rs` already? :)